tests/functional/deployment/test_fdf_standalone.py

```
@@ -0,0 +1,345 @@
+        idms_ocp = OCP(kind="ImageDigestMirrorSet")
+        try:
+            idms_data = idms_ocp.get()
+        except CommandFailed:
+            pytest.skip("ImageDigestMirrorSet CRD not available on this cluster")
+
+        items = idms_data.get("items", [])
+        logger.assertion(
+            f"IDMS resources: count={len(items)}, " f"has_resources={bool(items)}"
+        )
+        assert items, (
+            "No ImageDigestMirrorSet found — " "FDF image mirroring not configured"
```

Ruff (0.16.2)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)